public/client/library.js

```diff
@@ -24,15 +24,66 @@ function toggleToolTips(tipId) {
 }
 
 // toggle modal dispay
-function clickRemove(userBookId, title) {
+function clickRemove(userBookId, title, condition, date) {
     var modalLabel = document.getElementById('modalLabel');
     modalLabel.innerHTML = "Confirm Book Removal";
     var modalBody = document.getElementById('modalBody');
-    modalBody.innerHTML = `Are you sure you want to remove <b>"${title}"</b> from your library?`
+    if(userBookId.length > 1){
+        var condition = condition.split(',');
```

Comment:
This modal should be created using bootstrap on the page and then dynamically populated using a call back to the server on button click. Because were short on time it's not worth fixing now but it's worth noting as a suggestion.